projects/common/vc709/vc709_system_constr.xdc

```diff
@@ -0,0 +1,55 @@
+
+# constraints
+
+set_property -dict  {PACKAGE_PIN  AV40  IOSTANDARD  LVCMOS18} [get_ports sys_rst]
+
+# clocks
+
+set_property -dict  {PACKAGE_PIN  H19   IOSTANDARD  DIFF_SSTL15} [get_ports sys_clk_p]
+set_property -dict  {PACKAGE_PIN  G18   IOSTANDARD  DIFF_SSTL15} [get_ports sys_clk_n]
+
+# uart
+
+set_property -dict  {PACKAGE_PIN  AU33  IOSTANDARD  LVCMOS18} [get_ports uart_sin]
+set_property -dict  {PACKAGE_PIN  AU36  IOSTANDARD  LVCMOS18} [get_ports uart_sout]
+
+# fan
+
+set_property -dict  {PACKAGE_PIN  BA37  IOSTANDARD  LVCMOS18} [get_ports fan_pwm]
+
+set_property -dict  {PACKAGE_PIN  AV30  IOSTANDARD  LVCMOS18} [get_ports gpio_bd[0]]    ; ## GPIO_DIP_SW0
+set_property -dict  {PACKAGE_PIN  AY33  IOSTANDARD  LVCMOS18} [get_ports gpio_bd[1]]    ; ## GPIO_DIP_SW1
+set_property -dict  {PACKAGE_PIN  BA31  IOSTANDARD  LVCMOS18} [get_ports gpio_bd[2]]    ; ## GPIO_DIP_SW2
+set_property -dict  {PACKAGE_PIN  BA32  IOSTANDARD  LVCMOS18} [get_ports gpio_bd[3]]    ; ## GPIO_DIP_SW3
+set_property -dict  {PACKAGE_PIN  AW30  IOSTANDARD  LVCMOS18} [get_ports gpio_bd[4]]    ; ## GPIO_DIP_SW4
+set_property -dict  {PACKAGE_PIN  AY30  IOSTANDARD  LVCMOS18} [get_ports gpio_bd[5]]    ; ## GPIO_DIP_SW5
+set_property -dict  {PACKAGE_PIN  BA30  IOSTANDARD  LVCMOS18} [get_ports gpio_bd[6]]    ; ## GPIO_DIP_SW6
+set_property -dict  {PACKAGE_PIN  BB31  IOSTANDARD  LVCMOS18} [get_ports gpio_bd[7]]    ; ## GPIO_DIP_SW7
+set_property -dict  {PACKAGE_PIN  AR40  IOSTANDARD  LVCMOS18} [get_ports gpio_bd[8]]    ; ## GPIO_SW_N
+set_property -dict  {PACKAGE_PIN  AU38  IOSTANDARD  LVCMOS18} [get_ports gpio_bd[9]]    ; ## GPIO_SW_E
+set_property -dict  {PACKAGE_PIN  AP40  IOSTANDARD  LVCMOS18} [get_ports gpio_bd[10]]   ; ## GPIO_SW_S
+set_property -dict  {PACKAGE_PIN  AW40  IOSTANDARD  LVCMOS18} [get_ports gpio_bd[11]]   ; ## GPIO_SW_W
+set_property -dict  {PACKAGE_PIN  AV39  IOSTANDARD  LVCMOS18} [get_ports gpio_bd[12]]   ; ## GPIO_SW_C
+set_property -dict  {PACKAGE_PIN  AM39  IOSTANDARD  LVCMOS18} [get_ports gpio_bd[13]]   ; ## GPIO_LED_0_LS
+set_property -dict  {PACKAGE_PIN  AN39  IOSTANDARD  LVCMOS18} [get_ports gpio_bd[14]]   ; ## GPIO_LED_1_LS
+set_property -dict  {PACKAGE_PIN  AR37  IOSTANDARD  LVCMOS18} [get_ports gpio_bd[15]]   ; ## GPIO_LED_2_LS
+set_property -dict  {PACKAGE_PIN  AT37  IOSTANDARD  LVCMOS18} [get_ports gpio_bd[16]]   ; ## GPIO_LED_3_LS
+set_property -dict  {PACKAGE_PIN  AR35  IOSTANDARD  LVCMOS18} [get_ports gpio_bd[17]]   ; ## GPIO_LED_4_LS
+set_property -dict  {PACKAGE_PIN  AP41  IOSTANDARD  LVCMOS18} [get_ports gpio_bd[18]]   ; ## GPIO_LED_5_LS
+set_property -dict  {PACKAGE_PIN  AP42  IOSTANDARD  LVCMOS18} [get_ports gpio_bd[19]]   ; ## GPIO_LED_6_LS
+set_property -dict  {PACKAGE_PIN  AU39  IOSTANDARD  LVCMOS18} [get_ports gpio_bd[20]]   ; ## GPIO_LED_7_LS
```

Comment:
These signals do not exist in the system_top.v ; Update it with gpio_bd_i or gpio_bd_o

Reply:
Done in v2.